hyper/http11/response.py

```diff
@@ -48,7 +49,8 @@ def __init__(self, code, reason, headers, sock, connection=None):
         # bother checking for content-length, we just keep reading until
         # we no longer can.
         self._expect_close = False
-        if b'close' in self.headers.get(b'connection', []):
+        if (b'close' in self.headers.get(b'connection', []) or
+                b'close' in self.headers.get(b'proxy-connection', [])):
             self._expect_close = True
```

Comment:
I mean, the above response is invalid: Proxy-Connection is not a HTTP/1.1 header that has any effect on framing, so that assertion really should be triggered.